tutorials/getting_started/first_steps.md

```diff
@@ -31,13 +31,18 @@ Start by setting your language, region, and time zone. You can also set your loc
 ### Install add-ons
 
 You now have the option to install openHAB add-ons.
-Do this if you already know that you'll need specific add-ons (e.g. if you're upgrading from an older openHAB system), or skip this step if you prefer to install add-ons individually.
+The openHAB system does an automatic scan to discover home automation equipment and software on your PC and LAN, and based the result it recommends suggested add-ons to install. 
```

Comment:
"and based the result it recommends suggested add-ons to install." - I think there is something wrong with that sentence, IMHO

Either 
"and based >on< the result it recommends suggested add-ons to install."

or
"and suggests installing add-ons based on the result."